test/test_mmnet.py

```diff
@@ -15,7 +15,7 @@
 from features import *
 
 
-class TestPeopleImport(unittest.TestCase):
+class TestNetworkImport(unittest.TestCase):
```

Comment:
Better to have two test classes, one for the people import and one for the events.